.rubocop.yml

```diff
@@ -21,6 +21,9 @@ Style/SpaceInsideHashLiteralBraces:
 Style/FileName:
   Enabled: false
 
+Metrics/AbcSize:
+  Max: 20
+
```

Comment:
Also, I'd suggest running `rubocop -aD`.  That will fix any issues that it can, and then you can resolve the rest.  The `D` will output the cop's name so that you can look it up in case you don't know what it's talking about ;)